src/EnergyPlus/AirLoopHVACDOAS.hh

```diff
@@ -174,7 +174,7 @@ namespace AirLoopHVACDOAS {
         // default constructor
         AirLoopDOAS() // constructor
             : SumMassFlowRate(0.0), PreheatTemp(-999.0), PrecoolTemp(-999.0), PreheatHumRat(-999.0), PrecoolHumRat(-999.0), SizingMassFlow(0.0),
-              SizingCoolOATemp(-999.0), SizingCoolOAHumRat(-999.0), HeatOutTemp(0.0), HeatOutHumRat(0.0), m_AirLoopDOASNum(0), m_OASystemNum(0),
+              SizingCoolOATemp(-999.0), SizingCoolOAHumRat(-999.0), HeatOutTemp(999.0), HeatOutHumRat(999.0), m_AirLoopDOASNum(0), m_OASystemNum(0),
```

Comment:
Since looking for max T for cooling and min T for heating, the heating initialization was changed to allow comparison to outdoor T (i.e., if OAT > SizingCoolOATemp then save cooling OA T and w, and if OAT < HeatOutTemp then save heating OA T and w).